pyproject.toml

```diff
@@ -39,7 +39,8 @@ nautobot-plugin-nornir = ">=1.0.0"
 
 [tool.poetry.dev-dependencies]
 bandit = "*"
-black = "*"
+# Black 23.x.x configuration changes and migration files are taken into account
```

Comment:
I hate doing this, but why do we want this? won't this hold our version of black in forever? I think we can take the one time hit vs that.

Reply:
Usually, we leave the dev dependencies "open" when the plugin is "baked", and then we pin them to avoid surprises like the one I found here.
If we want to keep this open, then, in this PR, I should also fix the black configuration. Not a problem at all, but I wanted to keep it simple, and leave this change to another PR, if desired.

Reply:
How much work is it to get black passing when using the updated version?  I'd tend to like to just get it taken care of at the same time.

Reply:
No idea, not even considered to mix both things in a PR

Reply:
However, if necessary I could do it, for sure.

Reply:
@jeffkala @itdependsnetworks final thought?